### ISSUE TITLE
Generate playlist out of liked/saved songs from user

### ISSUE THREAD

**Issue by bardimardy** [rtkg12/remixr#8]
Add the opportunity to generate a playlist out of liked/saved songs from the user.
The API should provide the possibillity to get these data.

Would take on this task to implement this feature, my first PR on Hacktoberfest :P
Please share your opinion on this idea.

**Comment by ashikvarma11** [comment]
@bardimardy  Good idea. I'm working #2 . Probably will make a PR in a couple of days. I believe this is dependent on that.

**Comment by rtkg12** [comment]
@bardimardy That's a good idea, but could you elaborate on that? It might be useful for some users, but not for me personally because I have thousands of songs saved in my Spotify library spanning a wide range of genres. So, a playlist built on that won't be very effective, but there might be some value in doing that.

**Comment by bardimardy** [comment]
@rtkg12 Yea of course, I would start to work on that the next days. Got this idea while I was looking if this function already exists because the most of my songs are only saved in the favourites. 😄

**Comment by Emeka-Ugbanu-hub** [comment]
I would love to help with this
